<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientMapFilterConfigWidget</class>
 <widget class="QWidget" name="GradientMapFilterConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gradient Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGradientColors">
      <attribute name="title">
       <string>Gradient Colors</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollAreaGradientColors">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="KisGenericGradientEditor" name="widgetGradientEditor">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>485</width>
            <height>236</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabColorMode">
      <attribute name="title">
       <string>Color Mode</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QComboBox" name="comboBoxColorMode">
         <item>
          <property name="text">
           <string>Blend</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Nearest</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dither</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidgetColorMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <property name="prefix" stdset="0">
          <string>Amount:  </string>
         </property>
         <widget class="QWidget" name="blendPage">
          <layout class="QFormLayout" name="formLayout_7"/>
         </widget>
         <widget class="QWidget" name="nearestPage">
          <layout class="QFormLayout" name="formLayout_6"/>
         </widget>
         <widget class="QWidget" name="ditherPage">
          <layout class="QFormLayout" name="formLayout_5">
           <item row="0" column="0" colspan="2">
            <widget class="KisDitherWidget" name="widgetDither" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisDitherWidget</class>
   <extends>QWidget</extends>
   <header location="global">KisDitherWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisGenericGradientEditor</class>
   <extends>QWidget</extends>
   <header location="global">KisGenericGradientEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>comboBoxColorMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidgetColorMode</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stackedWidgetColorMode</sender>
   <signal>currentChanged(int)</signal>
   <receiver>comboBoxColorMode</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
